src/fastfetch.c

```diff
@@ -439,7 +441,7 @@ static inline void printCommandHelp(const char* command)
             "Album name"
         );
     }
-    else if(ffStrEqualsIgnCase(command, "datetime-format") == 0 || ffStrEqualsIgnCase(command, "date-format") || ffStrEqualsIgnCase(command, "time-format"))
+    else if(ffStrEqualsIgnCase(command, "datetime-format") || ffStrEqualsIgnCase(command, "date-format") || ffStrEqualsIgnCase(command, "time-format"))
```

Comment:
We can remove `|| ffStrEqualsIgnCase(command, "date-format") || ffStrEqualsIgnCase(command, "time-format")` as the modules `date` and `time` no longer exist.